sdk/resourcemanager/azure-resourcemanager-servicebus/README.md

```diff
@@ -22,10 +23,46 @@ For documentation on how to use this package, please see [Azure Management Libra
 ```
 [//]: # ({x-version-update-end})
 
+### Include the recommended packages
+
+Azure Management Libraries require a `TokenCredential` implementation for authentication and an `HttpClient` implementation for HTTP client.
+
+[Azure Identity][azure_identity] package and [Azure Core Netty HTTP][azure_core_http_netty] package provide the default implementation.
+
+### Authentication
+
+By default, Azure Active Directory token authentication depends on correct configure of following environment variables.
+
+- `AZURE_CLIENT_ID` for Azure client ID.
+- `AZURE_TENANT_ID` for Azure tenant ID.
+- `AZURE_CLIENT_SECRET` or `AZURE_CLIENT_CERTIFICATE_PATH` for client secret or client certificate.
+
+In addition, Azure subscription ID can be configured via environment variable `AZURE_SUBSCRIPTION_ID`.
+
+With above configuration, `azure` client can be authenticated by following code:
```

Comment:
maybe it is not "`azure` client".

Reply:
Here in these samples, I still prefer short `azure` var instead of full `azureResourceManager`.